File: second_semister/oop/week_03/exam/hello.js
let newArr = []

function hello(arr, newArr, i) {
    if (arr.length === i) {
        // resutl = hello(arr, newArr, 0);
        // return result
        return
    }
    newArr.push(arr[i])
    resutl = hello(arr, newArr, i + 1);
    return resutl
}

console.log(hello([1, 2, 3], newArr, 0));

File: firs_semister/c/week_10/files/hello.c
#include <stdio.h>
#include <stdlib.h>
// gcc hello.c -o hello && ./hello < input.txt  > output.txt
int main()
{
    
    FILE *inputFile;

    int n = 5;
    int sum = 0;
    int arr[n];
    scanf("%d", &n);
    for (int i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }
    for (int i = 0; i < n; i++)
    {
        sum += arr[i];
    }
    printf("sum -> %d\n", sum);
    return 0;
}